test/runtest.py

```diff
@@ -974,9 +1036,13 @@ def testResourceConstraints(self):
         def get_worker_id():
             time.sleep(1)
             return sys.path[-1]
+
         while True:
-            if len(set(ray.get([get_worker_id.remote()
-                                for _ in range(num_workers)]))) == num_workers:
+            if len(
```

Comment:
This (and the one below) looks a little ugly, but most of the other formating changes look like improvements to me; we can just rewrite the code below next time we touch it ;)

Reply:
SG.  I don't have a preference of style but definitely think consistency is best.  I'd suggest everyone to check out `yapfify` & its integration in your editor of choice.  We can tweak the style file later (it's now the default generated template).